Sources/UInt128.swift

```diff
@@ -108,7 +108,7 @@ public struct UInt128 {
 extension UInt128 : FixedWidthInteger {
     // MARK: Instance Properties
 
-    public static var bitWidth : Int { return 128 }
+    public static var bitWidth : Int { return UInt64.bitWidth * 2 }
```

Comment:
I would prefer to have the fixed length instead of a computed one. The library is for UInt128, so it should always be 128 bits wide.